### PR TITLE
Update bootstrap typography

### LINKED ISSUE #10841: Bootstrap theme typography enhancements
### Is your feature request related to a problem? Please describe.
I noticed that in the Bootstrap theme typography doesn't match the original bootstrap component typography as shown in the bootstrap repo.

### PR DESCRIPTION
**Pull request by desig9stein**
Closes #10841 

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss [hunk 1]
```diff
@@ -133,6 +132,32 @@
...
+            text: 'body-1'
+        ));
+
+        @include igx-drop-down-typography($type-scale, $categories: (
+            header: 'body-2',
+            select-item: 'body-2'
+        ));
+
+        @include igx-expansion-panel-typography($type-scale, $categories: (
+            description: 'body-2',
```
Comment (StefanIvanov) [inline projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss:159]:
```suggestion
            description: 'body-1',
```

File: projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss [hunk 2]
```diff
@@ -133,6 +132,32 @@
...
+        ));
+
+        @include igx-navbar-typography($type-scale, $categories: (
+            title: 'h5'
+        ));
+
+        @include igx-toast-typography($type-scale, $categories: (
+            text: 'body-1'
+        ));
+
```
Comment (StefanIvanov) [inline projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss:152]:
```suggestion
```
should be body 2 as in the default theme

File: projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss [hunk 3]
```diff
@@ -133,6 +132,32 @@
...
+        @include igx-drop-down-typography($type-scale, $categories: (
+            header: 'body-2',
+            select-item: 'body-2'
```
Comment (StefanIvanov) [inline projects/igniteui-angular/src/lib/core/styles/typography/_typography.scss:155]:
```suggestion
            header: 'body-1',
            item: 'body-1',
            select-item: 'body-1'
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]
Could you also make the h6 to have font width 500?

**Review by StefanIvanov — APPROVED** [review]

**Review by simeonoff — APPROVED** [review]

**Pull request merged**